src/adapter/src/coord.rs

```diff
@@ -386,8 +386,7 @@ pub enum PeekStage {
     LinearizeTimestamp(PeekStageLinearizeTimestamp),
     RealTimeRecency(PeekStageRealTimeRecency),
     TimestampReadHold(PeekStageTimestampReadHold),
-    OptimizeMir(PeekStageOptimizeMir),
-    OptimizeLir(PeekStageOptimizeLir),
+    OptimizeMir(PeekStageOptimize),
```

Comment:
Should this variant just be called `Optimize`?

Reply:
Yes.